### DISCUSSION TITLE
Timeline Plugin starting from a custom number

### DISCUSSION THREAD

**Discussion by S3TTE** [katspaugh/wavesurfer.js#3305] [Q&A] [answered]
I want to use the timeline plugin under a waveform having the starting number not equal to 0 but as a custum numer or better a custom datetime.

for example
this is what i got from the documentation
![image](https://github.com/katspaugh/wavesurfer.js/assets/143178970/9dacff2a-137e-4706-995f-65b00638697a)
but i want somethin like that
![image](https://github.com/katspaugh/wavesurfer.js/assets/143178970/5c2cb486-de90-4916-822f-f16c0c1f4bb5)
or even better
![image](https://github.com/katspaugh/wavesurfer.js/assets/143178970/f59f2c37-2d6a-4b6a-b207-dece86b0a37d)

is it possible? how can i do?

so i just want to edit the label of the timeline, so the user can understand for exaple that the waveform loaded is a wave from the 12:00AM

here the code i'm using but it doesn't work:

```
var waveform = WaveSurfer.create({
        container: "#waveform",
        waveColor: "#4F4A85",
        progressColor: "#383351",
        barWidth: 2,
        sampleRate: 14600,
        plugins: [TimelinePlugin.create({
          height: 10,
          timeInterval: 0.2,
          formatTimeCallback: "hh:mm",
          primaryLabelInterval: 1, 
          offset: 15,
          style: {
            fontSize: "10px",
            color: '#6A3274', 
          }
        })
      ]
      });
```
(i'm using the version 7.3.2)

**Comment by katspaugh** [comment]
You can do it via the `formatTimeCallback` option.

**Reply by S3TTE** [reply]
can u make a better example on how to use that option? I added a code in the question
thanks.

**Reply by katspaugh** [reply, marked as answer]
Docs: https://wavesurfer.xyz/docs/types/plugins_timeline.TimelinePluginOptions

E.g.
```js
formatTimeCallback: (seconds) => {
  const offsetTime = seconds + 15
  return Math.floor(offsetTime / 60) + ':' + offsetTime % 60
}
```

**Reply by S3TTE** [reply]
thanks
